version.properties

```diff
@@ -0,0 +1,4 @@
+#Version of the produced binaries. This file is intended to be checked-in.
+#It will be automatically bumped by release automation.
+version=1.0.0
+previousVersion=0.0.1
```

Comment:
We discussed offline that we should keep the previous version of this?

Reply:
fixed